src/erlcloud_sns.erl

```diff
@@ -487,11 +489,11 @@ get_notification_attribute(Attribute, Notification) ->
 
 
 
--spec(set_topic_attributes/3 :: (sns_topic_attribute_name(), string(), string()) -> ok).
+-spec(set_topic_attributes/3 :: (sns_topic_attribute_name(), parameter(), parameter()) -> ok).
 set_topic_attributes(AttributeName, AttributeValue, TopicArn) ->
     set_topic_attributes(AttributeName, AttributeValue, TopicArn, default_config()).
```

Comment:
Line 494 is too long:     set_topic_attributes(AttributeName, AttributeValue, TopicArn, default_config())..